content/blog/2023-03-14-mining-ethereum-classic-with-an-ipollo-asic-through-2miners-pool/index.md

```diff
@@ -0,0 +1,178 @@
+
+For this, go to the “Password Change” tab on the left menu, then enter the new password and press “Save”.
+
+![Change password.](./6.png)
+
+## 7. Set Up Your Miner Credentials
+
+To set up your miner credentials (the configuration data so your iPollo miner will be pointing to your account on the 2miners pool) go to the “Miner Configuration” tab on the left menu.
+
+Once on that screen, change the default mining algorithm from ETHASH to ETCHASH on the dropdown menu at the center top of the page.
+
+![Miner configuration.](./7.png)
+
+## 8. Get Miner Settings Information From The 2miners Mining Pool
+
+Now you need to go to 2miners to get your credentials or configuration data. For this, go to:
+
+https://etc.2miners.com/help 
+
+Once there, create an ETC wallet as per the instructions or use an existing wallet you already have, if you wish.
+
+Then, get the instruction shown further below in that page for “ASIC Settings”. The instructions are:
+
+**URL: stratum+tcp://etc.2miners.com:1010**
+
+**Worker: YOUR_WALLET_ADDRESS.ASIC_ID**
+
+**Password: x**
+
+![Miner settings from 2miners.](./8.png)
+
+## 9. Enter Your Configuration on Your iPollo
+
+Now, you have to go back to the iPollo ASIC miner app and enter your configuration in the “Miner Configuration” page. In our example, we used the account shown below with the extension “ASIC_ID”, but you may use any ID number like ASIC_1 or ASIC_28, etc. The ID is in case you have several miners pointing to 2miners so you may identify them independently
+
+Enter your configuration in the fields provided as shown in the image and press “Save & Apply”. Our miner configuration data in this example was: 
+
+**Pool 1: stratum+tcp://etc.2miners.com:1010**
+
+**Worker: 0x69f0da9aC5196e8817af39a2F1E0d43209946039.ASIC_ID**
+
+**Password: x**
+
+![Enter configuration data.](./9.png)
+
+## 10. Check the Stats for Your Mining Rig
+
+Once you saved and applied your miner configuration data, your iPollo ASIC rig will start mining for 2miners. You may go to the “Running Overview” tab on the left menu to see the activity of your miner live.
+
+You must wait 5 to 10 minutes for your iPollo to talk to 2miners and establish a connection. After a while, you will see that the line in the chart will start to go up, increasing the hashrate that is pointing to your 2miners account. You will also see other stats as the temperature of your machine, the RPM of the fan, and the current hashrate output of your iPollo.
+
+![Check mining stats.](./10.png)
+
+## 11. See if the Work Is Being Accepted
+
+Then, you will want to see if the work of your iPollo is actually being accepted by the 2miners mining pool. For this, go to the “Information Overview” tab on the left menu to see the statistics.
+
+In our example, the device number “0” (our machine) is enabled and active. For the URL which it is pointed to, it has performed 44 works of which 8 have been accepted, and 6:38 minutes have elapsed since we started mining.
```

Comment:
Line 154: Correct structure
```suggestion
In our example, the device number “0” (our machine) is enabled and active. For the URL to which it is pointed, it has performed 44 works of which 8 have been accepted, and 6:38 minutes have elapsed since we started mining.
```